### PR TITLE
Feature/add jenkinks link

### PR DESCRIPTION
**Pull request by devshorts**
Doing refactoring on typescript files and adding jenkins link next to jobs for easy configuration:

image

### CODE REVIEW

File: core/src/main/resources/assets/francois/services/francois.ts [hunk 1]
```diff
@@ -17,35 +41,35 @@ export class FrancoisApi {
...
             headers: {
                 'Content-Type': 'application/json'
             }
         });
     }
 
-    createJob(templateName: string, job: JobApplication) {
-        return this.http.post(`/api/v1/francois/templates/${templateName}/jobs`, JSON.stringify(job), {
+    createJob(templateName:string, job:JobApplication) {
+        return this.http.post(`/api/v1/francois/templates/${templateName}/jobs`, JSON.stringify(job), <ngHttp.RequestOptionsArgs>{
```
Comment (jakeswenson) [inline core/src/main/resources/assets/francois/services/francois.ts:65]: is the cast necessary?

File: core/src/main/resources/assets/francois/views/jenkins_iframe.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<iframe name="_jenkins" src="{{currentJenkinsUrl}}"></iframe>
```
Comment (jakeswenson) [inline core/src/main/resources/assets/francois/views/jenkins_iframe.html:1]: is this working? lets remove if not

**Comment by jakeswenson** [comment]
TYTE.

File: core/src/main/resources/assets/francois/services/francois.ts (hunk 1 above)
Comment (devshorts) [inline core/src/main/resources/assets/francois/services/francois.ts:65]: You know, I don't know. IntelliJ kept bitching at me and when I did that it stopped being red ¯_(ツ)_/¯. Functionally it made no difference though

**Pull request merged**